### DISCUSSION TITLE
Inviting past presenters to BHNT100 Oct 29 19:00

### DISCUSSION THREAD

**Discussion by lucasrangit** [berlin-hack-and-tell/bhnt.c-base.org#462] [General]
Hi! We're celebrating the 100 edition of BHNT on October 29th at 19:00 and would like past presenters to be there. You and your hacks are the vital force of our meetup. Come so we can thank you in person! See you soon 🙏🏽  

RSVP: https://bhnt.c-base.org/2024-10-29-no100-annivercary

@030jmk
@0xaverageuser
@0xramsi
@22nds
@acebusters
@akrennmair
@algroznykh
@alvaro-c
@andrewtavis
@archgpt
@arrybn
@artfwo
@arthurtent
@bashtian
@benjaminwand
@bergie
@bplevin36
@bubu
@cesalazar
@cesena
@cfcs
@cfretter
@chemaclass
@chr5tphr
@chrischinchilla
@chriseth
@ciex
@comcomist
@crest
@ctrlw
@cyroxx
@d1plo1d
@d3d9
@danielpaulus
@daniel-schuermann
@dashersw
@davedarko
@davidblitz
@davidspeith
@dcarral
@devmachiine
@dkundel
@dpgn
@eliasdoehne
@excds
@eyedol
@fassko
@floris-vos
@frehberg
@friedrichwessel
@fuzxxl
@georgjaehnig
@gimlet2
@gnunicorn
@gorgiastro
@hartwork
@hasanshaukat
@hayeah
@heartsucker
@herrmuellerluedenscheid
@ianklatzco
@ihucos
@interkosmos
@isaksolheim
@janek
@janlelis
@jarek
@jaseg
@jcbevns
@jmcq87
@johnjohndoe
@jonas-braun
@jpeinelt
@jpetazzo
@justinkelly69
@jvirtanen
@jzilg
@kaeruct
@kaihendry
@kajogo777
@karlb
@kevinveenbirkenbach
@k-nut
@langa-me
@lavielle
@leandrosansilva
@leif
@leopic
@lgrammel
@lordofthelake
@lucasrangit
@maggo
@masseast
@meowmeowxw
@miceuz
@michaelkreil
@michibo
@miserlou
@mknz
@mmabrouk
@momonala
@mwarning
@neopostmodern
@nikkyai
@nolash
@okwme
@pan-maruda
@pdp7
@phreeza
@pixel16
@pkoryzna
@planemad
@ppurang
@ptef
@quesada
@quittymr
@rastapasta
@remarcable
@ri0t
@richelleji
@ricott1
@rlods
@roryscarson
@rtens
@sargsyan
@schneeschni
@soulim
@sp4ceberlin
@sriemer
@ssimono
@steely-glint
@stefanmedack
@strathausen
@stvad
@sudoaza
@surma
@szat
@tasomaniac
@tatocaster
@tgalal
@timdaub
@timwi
@ugogon
@uwekamper
@vog
@vrde
@weliveindetail
@whalecoiner
@wikinaut
@wleuschner
@wolframhempel
@wrl
@yncyrydybyl

**Comment by cafca** [comment]
Is it Oct 29 at 7pm?

**Reply by lucasrangit** [reply]
Yes, hope to see you there! Sorry for leaving out this important detail. I added it to the description.

**Comment by ChrisChinchilla** [comment]
Cool! For some reason I haven't seen any Meetup notifications about the group for ages (probably meetup being rubbish), so look forward to it!

**Comment by dashersw** [comment]
Oh this sounds awesome! I miss this community so much :) had some of the best times of my life. I'll definitely be there!

**Comment by altsoph** [comment]
I'm not in the list above, but I was the HotM @ BHNT42 and presented a couple of things later. How can I RSVP? :)

**Comment by ligi** [comment]
somehow BHNT 42 does not have any data here:
https://bhnt.c-base.org/2016-02-23-no42-hackers-guide-to-the-galaxy
And this is the source of this list.
Feel free to PR details about the event.
RSVPs did not yet open - should be done soon.

**Reply by altsoph** [reply]
Done: https://github.com/berlin-hack-and-tell/bhnt.c-base.org/pull/463/files

**Reply by ligi** [reply]
merged

**Comment by ligi** [comment]
meetup announced - please RSVP and bring your hacks! https://bhnt.c-base.org/2024-10-29-no100-annivercary
